uber/templates/closed.html

```diff
@@ -3,7 +3,7 @@
 {% block backlink %}{% endblock %}
 {% block content %}
 
-<h2 style="color:red">The {{ c.EVENT_NAME }} registration/volunteer wesite is temporarily offline</h2>
+<h2 style="color:red">The {{ c.EVENT_NAME }} registration/volunteer website is temporarily offline</h2>
```

Comment:
I like to think of this as not "my site" but "our site", for which "wesite" is a common abbreviation :P

All joking aside, +1